packages/devtools/src/commands/switchToFrame.ts

```diff
@@ -22,18 +31,18 @@ export default async function switchToFrame ({ id }) {
         while (parentFrame) {
             parentFrame = await parentFrame.parentFrame()
         }
-        this.currentFrame = parentFrame
+        this.currentFrame = parentFrame as unknown as Page
         return null
     }
 
     /**
      * switch frame by element ID
      */
-    if (typeof id[ELEMENT_KEY] === 'string') {
-        const elementHandle = await this.elementStore.get(id[ELEMENT_KEY])
+    if (typeof (id as unknown as WebdriverIO.ElementReference)[ELEMENT_KEY] === 'string') {
```

Comment:
Let's define a variable for proprly typed id instead of casting 2 times to improve readability